frontend/src/marketplace/EntryList.tsx

```diff
@@ -1,6 +1,6 @@
 import { EntrySearchResult } from 'actions/model'
-import { CSSProperties } from 'react'
-import { FixedSizeList } from 'react-window'
+import { CSSProperties, useLayoutEffect, useState } from 'react'
+import { VariableSizeList as List } from 'react-window'
```

Comment:
I'm not a fan of renaming the component like this. At a glance it's not obvious that it's not a MUI `List`

Reply:
This is how it's done in all the docs, but I agree it's confusing, so reverted.